test/com/gfredericks/test/chuck/properties_test.cljc

```diff
@@ -0,0 +1,49 @@
+(ns com.gfredericks.test.chuck.properties-test
+  #?(:cljs (:require-macros [com.gfredericks.test.chuck.properties.macros :as prop']
+                            [com.gfredericks.test.chuck.generators :as gen']))
+  #?(:clj  (:require [clojure.test :refer :all]
+                     [clojure.test.check :as t.c]
+                     [clojure.test.check.generators :as gen]
+                     [clojure.test.check.clojure-test :refer [defspec]]
+                     [com.gfredericks.test.chuck.properties :as prop'])
+     :cljs (:require [cljs.test :refer-macros [run-tests deftest is]]
+                     [cljs.test.check :as t.c]
+                     [cljs.test.check.generators :as gen]
+                     [cljs.test.check.properties :include-macros true]
+                     [cljs.test.check.cljs-test :refer-macros [defspec]])))
```

Comment:
`0.8.0-RC3` is in the oven and should be available within 20 minutes or so. I fixed the problem with latest cljs, so you shoud be able to use `1.7.48` as well.